src/pages/Home.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+import { useState } from 'react';
+
+import Navigation from '../components/Navbar';
+import List from '../components/List';
+import Form from '../components/Form';
+
+function HomePage() {
+  const [books, setBooks] = useState([]);
+
+  const handleAddBook = (book) => {
+    setBooks((prevBooks) => [...prevBooks, { id: Math.random(), ...book }]);
+  };
+
+  const handleDeleteBook = (id) => {
+    setBooks((prevBooks) => prevBooks.filter((book) => book.id !== id));
+  };
```

Comment:
- [ ] You have done a great job implementing the functionality to add and delete a book :clap: But, those functionalities are not required in this milestone therefore I can not review them. 
  However, I suggest you add a list of books to the initial state that will be rendered when the application loads. It could look like this :point_down: 
```js
  const [books, setBooks] = useState([
  {
    title: 'book 1',
    author: 'author 1',
  },
  ...
]);
```